src/orderly/cli/__init__.py

```diff
@@ -0,0 +1,204 @@
+import sys
+from collections import Counter
+
+import click
+
+import outpack.search
+from orderly.cli.options import (
+    NumericalParamType,
+    PropagatingGroup,
+    with_root,
+    with_search_options,
+)
+from orderly.run import orderly_run
+from outpack.init import outpack_init
+from outpack.location import (
+    outpack_location_add,
+    outpack_location_list,
+    outpack_location_remove,
+    outpack_location_rename,
+)
+from outpack.util import format_list
+
+
+@click.group(cls=PropagatingGroup)
+@with_root(propagate=True)
+def cli():
+    pass
+
+
+@cli.command()
+@click.argument("path")
+@click.option(
+    "--archive", help="Path to the archive in which packets are stored."
```

Comment:
This renders the help as `--archive TEXT` which is confusing - do you need to override something?

Reply:
Ah yeah good catch, can set a metavar for it. Surprised it doesn't already do that.